recipes-ni/ni-utils/files/x64/fw_printenv

```diff
@@ -43,7 +43,7 @@ show_variable_grub(){
 			key=$(echo "$line" | cut -d "=" -f 1)
 			#if the $1 matches exactly the key, extract the value
 			if [ "$key" = "$1" ] ; then
-				value_tmp=$(echo "$line" | cut -d "=" -f 2)
+				value_tmp=$(echo "$line" | cut -d "=" -f 2-)
```

Comment:
Does the ARM version work as is for these new `othbootargs`? Or is it not applicable in ARM?
Also, I just realized that there is a version of this file in [os-common](https://p4.natinst.com/browser/penguin/nilinux/os-common/trunk/8.8/source/arch/x64/fw_printenv). And, I'm not sure which one of these get installed to the target.

Reply:
ARM uses the u-boot version of fw_printenv/setenv which doesn't have this problem.

I've noticed the os-common version too but I don't see it referenced in any of the makefiles. I have a p4 change ready for os-common with this change but I was waiting to see if it's necessary or if that version is dead code.